.drone.star

```diff
@@ -804,5 +806,5 @@ def versionize(version):
   if 'behat_version' in version:
     raw_version = version['behat_version']
   else:
-    raw_version = version['value']
-  return 'v%s' % (raw_version.replace("rc", "RC").replace("-", ""))
+    raw_version = 'v%s' % (version['value'].replace("rc", "RC").replace("-", ""))
+  return raw_version
```

Comment:
Note: `versionize()` is actually only ever called to determine the tag or branch from which to get the Behat `composer.json` for the acceptance tests. So changes to this method have no risk of changing the actual binary docker image created.

The new logic here is:
- If a specific `behat_version` has been specified, then use that (do not add a `v`  to the front, do not remove `-`, do not change it at all) The caller can specify exactly what they want.
- else do the logic to "mangle" the `value` do get it into a form that (should) match the official tag in the core repo for the release.